.github/copilot-instructions.md

```diff
@@ -1,626 +1,59 @@
 # Lewee Development Instructions
 
-## Overview
-
-Lewee is an opinionated set of packages to assist with setting up a domain-driven design architecture within ASP.NET. This repository contains both the Lewee framework packages and a sample restaurant management application demonstrating their usage.
-
-## How to Use These Instructions
-
-**Priority Order:**
+This file provides an overview and links to detailed Copilot instructions for different aspects of development in this repository.
```

Comment:
Please add another paragragh asking to keep this file and the sub-instruction files up-to-date as code changes over time